test/controllers/stacks_controller_test.rb

```diff
@@ -54,11 +54,12 @@ class StacksControllerTest < ActionController::TestCase
       assert_redirected_to '/github/auth/github?origin=http%3A%2F%2Ftest.host%2F'
     end
 
-    test "current_user must be a member of Shipit.github_team" do
-      Shipit.stubs(:github_team).returns(shipit_teams(:cyclimse_cooks))
+    test "current_user must be a member of at least a Shipit.github_teams" do
+      session[:user_id] = shipit_users(:bob).id
+      Shipit.stubs(:github_teams).returns([shipit_teams(:shopify_developers), shipit_teams(:cyclimse_cooks)])
       get :index
       assert_response :forbidden
-      assert_equal 'You must be a member of cyclimse/cooks to access this application.', response.body
+      assert_equal 'You must be a member of an authorized team to access this application.', response.body
```

Comment:
I'll add explicit tests for `Shipit.github_teams` to check for no team, `team` and both `team` and `teams`.